atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -271,13 +271,15 @@ public void compactInternally() {
                 log.info("Call to SHRINK SPACE COMPACT on table {} took {} ms.",
                         tableRef, shrinkAndCompactTimer.elapsed(TimeUnit.MILLISECONDS));
 
-                Stopwatch shrinkTimer = Stopwatch.createStarted();
-                conns.get().executeUnregisteredQuery(
-                        "ALTER TABLE " + oracleTableNameGetter.getInternalShortTableName(conns, tableRef)
-                                + " SHRINK SPACE");
-                log.info("Call to SHRINK SPACE on table {} took {} ms."
-                                + " This implies that locks on the entire table were held for this period.",
-                        tableRef, shrinkTimer.elapsed(TimeUnit.MILLISECONDS));
+                if (config.useShrinkCompactOnOracleStandardEdition()) {
```

Comment:
Feel like we should log 2 other things:
1) Log at the start of running this command noting that it may cause the table to lock with the name of the table.  This way if customers do see locking they know why.
2) We should log if this config is set to false that we aren't doing a full SHRINK and this may need to be run manually to free disk space.